jetty-http3/http3-client/src/main/java/org/eclipse/jetty/http3/client/internal/HTTP3SessionClient.java

```diff
@@ -106,6 +106,7 @@ private CompletableFuture<Stream> newRequest(long streamId, HeadersFrame frame,
             return promise;
 
         stream.setListener(listener);
+        notifyNewStream(listener, stream);
```

Comment:
I'd like this to be moved to `HTTP3StreamClient` via a new method `stream.onOpen()`, because `HTTP3StreamClient` has all the other `notifyXYZ()` methods.